tests/test_utils/tests.py

```diff
@@ -4,13 +4,13 @@
 import unittest
 import warnings
 
+from django import forms
```

Comment:
Create a separate pull request or commit in this pull request that precedes the changes for the ticket which makes the import change.